lib/charms/vault_k8s/v0/vault_autounseal.py

```diff
@@ -272,9 +283,11 @@ def __init__(self, charm: ops.CharmBase, relation_name: str):
         )
 
     def _on_relation_created(self, event: ops.RelationCreatedEvent) -> None:
+        """Handle the relation created event and emit a custom event."""
```

Comment:
You might want to document the ' event` arg here

Reply:
I'm not sure that would add any value.
I'd probably write something like:

```
Args:
   event: (RelationCreatedEvent) Triggered by Juju when event is created

```

Reply:
Well, it's up to you and the conventions you are following. I'm just asking :)